city_scrapers/spiders/newnj_nmc.py

```diff
@@ -0,0 +1,38 @@
+from city_scrapers_core.constants import CITY_COUNCIL
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import LegistarSpider
+
+
+class NewnjSpider(LegistarSpider):
+    name = "newnj_nmc"
+    agency = "Newark Municipal Council"
+    timezone = "America/Chicago"
+    start_urls = ["https://newark.legistar.com/Calendar.aspx"]
+    location = {
+        "address": "City Hall, 920 Broad Street, Newark, NJ 07102",
+        "name": "Municipal Council Chamber",
+    }
+
+    def parse_legistar(self, events):
+        """
+        `parse_legistar` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for event in events:
+            meeting = Meeting(
+                title=event["Name"]["label"],
+                description="",
+                classification=CITY_COUNCIL,
+                start=self.legistar_start(event),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self.location,
+                links=self.legistar_links(event),
+                source=self.legistar_source(event),
```

Comment:
Since you are utilizing the default method `legistar_source` from `LegistarSpider`, it will utilize the value of `item["Name"]["url"]`, which is more likely to be the URL of the department Municipal Council rather than a specific meeting URL. I suggest that we should instead use the value of `item["Meeting Details"]["url"]` for greater accuracy.

Reply:
Great catch, @lamle-ea. This is actually the first time I've scraped a Legistar site and made use of our Legistar spider so I wasn't aware of some of this handling. This is good to know and I've implemented your suggestion.